content/en/docs/collector/benchmarks.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Benchmarks
+weight: 99
+---
+  <style>
+
+    header {
+      width: 100%;
+      display: flex;
+      flex-direction: row;
+      background-color: rgb(79, 98, 173);
+    }
+
```

Comment:
We removed the header from the html, we can probably delete this and rely on the header css for opentelemetry.io

```suggestion
```